testZeroMessageRoundTrip expired

```sh
FAIL: testZeroMessageRoundTrip (__main__.RoundTripTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/grpc/grpc/python2.7_virtual_environment/lib/python2.7/site-packages/grpc/_adapter/_links_test.py", line 111, in testZeroMessageRoundTrip
    interfaces.BackToFrontTicket.Kind.COMPLETION)
AssertionError: <Kind.EXPIRATION: 'expiration'> is not <Kind.COMPLETION: 'completion'>

----------------------------------------------------------------------
Ran 5 tests in 3.379s
FAILED (failures=1)
D0602 19:08:45.137760445    9415 iomgr.c:106] Waiting for 1 iomgr objects to be destroyed
FAILED: grpc._adapter._links_test [ret=1, pid=9408]
```

This failed while trying to submit #1871

--- a/src/python/src/grpc/_adapter/_links_test.py
+++ b/src/python/src/grpc/_adapter/_links_test.py
@@ -40,7 +40,7 @@ from grpc.framework.base import interfaces
 from grpc.framework.foundation import logging_pool
 
 _IDENTITY = lambda x: x
-_TIMEOUT = 2
+_TIMEOUT = 32
 
 
 # TODO(nathaniel): End-to-end metadata testing.

